src/main/java/core/basesyntax/utility/FruitQuantityChecker.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.utility;
+
+import java.util.Map;
+
+public class FruitQuantityChecker {
+    public void checkFruitQuantity(Map<String, Integer> fruitMap) {
+        for (Integer temp : fruitMap.values()) {
```

Comment:
temp naming isn't applicable here
```suggestion
        for (Integer quantity : fruitMap.values()) {
```

Reply:
+